tasks.py

```diff
@@ -150,9 +170,10 @@ def docker_compose(context, command, **kwargs):
         command (str): Command string to append to the "docker compose ..." command, such as "build", "up", etc.
         **kwargs: Passed through to the context.run() call.
     """
+    CURRENT_GIT_BASE_BRANCH = get_current_git_base_branch(context)
     compose_command_tokens = [
         "docker compose",
-        f'--project-name "{context.nautobot.project_name}"',
+        f'--project-name "{context.nautobot.project_name}-{CURRENT_GIT_BASE_BRANCH}"',
```

Comment:
docker-compose doesn't like `ltm-1.6` as a part of the project name, so need to fix it up to have legal characters:

```suggestion
        f'--project-name "{context.nautobot.project_name}-{CURRENT_GIT_BASE_BRANCH.replace(".", "_")}"',
```